estime-api/src/main/java/com/estime/room/application/service/RoomApplicationService.java

```diff
@@ -85,18 +87,17 @@ public DateTimeSlotStatisticOutput calculateVoteStatistic(final Tsid roomSession
     }
 
     @Transactional(readOnly = true)
-    public Votes getParticipantVotesBySessionAndParticipantName(final Tsid roomSession, final String participantName) {
-        final Long roomId = getRoomIdBySession(RoomSession.from(roomSession));
-        final Long participantId = getParticipantIdByRoomIdAndName(roomId, participantName);
-        return voteRepository.findAllByParticipantId(participantId);
+    public ParticipantVotesOutput getParticipantVotesBySessionAndParticipantName(final VotesFindInput input) {
```

Comment:
❓ L3. 중요 질문 (Key Questions)
---

이제서야 보이는데 Participant가 Vote에 포함되니 아래와 같이 변경하는 것도 좋을 것 같아요
```suggestion
    public VotesOutput getVotesBySessionAndParticipantName(final VotesFindInput input) {
```

그렇게 되면 VotesFindInput이라는 파라미터가 자연스러워진다고 생각합니다~

Reply:
좋아요! 수정하겠습니다~!